docs/libcurl/curl_version_info.3

```diff
@@ -149,6 +149,9 @@ libcurl was built with support for TLS-SRP. (Added in 7.21.4)
 .IP CURL_VERSION_NTLM_WB
 libcurl was built with support for NTLM delegation to a winbind helper.
 (Added in 7.22.0)
+.IP CURL_VERSION_HSTS
+libcurl was built with support for HSTS
+(Added in 7.62.0)
```

Comment:
Probably 7.66.0 seems realistic now. Also applies to file `symbols-in-versions`.